django/applications/catmaid/control/volume.py

```diff
@@ -772,7 +772,7 @@ def export_volume(request, project_id, volume_id, extension):
             ', '.join(media_types), extension, ', '.join(acceptable[extension])), status=415)
     else:
         return HttpResponse('File type "{}" not understood. Known file types: {}'.format(
-            extension, ', '.join(acceptable.values())), status=415)
+            extension, ', '.join(*acceptable.values())), status=415)
```

Comment:
Pretty sure this is the right thing, but worth looking at

Reply:
I agree, the current error response is broken and needs to be fixed. While the unpacking you suggested would work for the current set of formats and media types, it would break when `acceptable` was something like:

```
acceptable = {
    'stl': ['model/stl', 'model/x.stl-ascii'], 'stl': ['model/stl', 'model/x.stl-ascii'],
    'obj': ['model/obj'],
}
```
How do you feel about the following:
```
from itertools import chain
acceptable_media_types = ', '.join(chain.from_iterable(acceptable.values())
```

Reply:
Either works for me (I think it'd be as valid to control what's allowed in the data structure as to make it more flexible). I'll modify it to your chain example.

Reply:
`', '.join(*acceptable.values())` is invalid, isn't it? `str.join` takes one iterable argument, I think, so as long as all of the values are strings the original `', '.join(acceptable.values())` is correct.

Tom's chain example works if all of the values are lists of strings, but am I right in thinking that the values can be either a string or a list of strings? Because strings are iterables, we'd end up with

```python
acceptable = {"stl": ['model/stl', 'model/x.stl-ascii'], "obj": "model/obj"}
types = ', '.join(chain.from_iterable(acceptable.values())
print(types)
```
giving us `"model/stl, model/x.stl-ascii, m, o, d, e, l, /, o, b, j"`.

`', '.join(chain.from_iterable([v] if isinstance(v, str) else v for v in acceptable.values())` should work for both!